app/src/main/java/com/ikarabulut/Game.java

```diff
@@ -2,9 +2,19 @@
 
 public class Game {
     private Board board;
+    private GameSettings gameSettings;
 
-    public Game(Board theBoard) {
+    public Game(Board theBoard, GameSettings theGameSettings) {
         board = theBoard;
+        gameSettings = theGameSettings;
+    }
+
+    public void startEvolutionLoops() {
+        int numOfEvolutions = gameSettings.getEvolutions();
+
+        for (int i=0; i<numOfEvolutions; i++) {
```

Comment:
I know we don't have the display part done yet, but how are you envisioning this working? Would you iterate over the number of evolutions and then print each evolution to the screen?

Reply:
Idk yet, that i what I am going to work on now.

I kinda wish I had a better understanding of Outside-in development, I thought I did but I was wrong, and now Im working in a pretty unorganized codebase.